<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEdit</class>
 <widget class="QWidget" name="TagEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horL">
     <item>
      <layout class="QVBoxLayout" name="foundedTagsLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Founded Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tag_search_result">
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolbar" name="toolbar" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Tag Editor</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Tag Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Alias</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListView" name="alias">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;s&gt;Tags&lt;/s&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QListView" name="tags">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTextEdit" name="comment">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionNewTag">
   <property name="icon">
    <iconset resource="uiresource.qrc">
     <normaloff>:/root/filenew.png</normaloff>:/root/filenew.png</iconset>
   </property>
   <property name="text">
    <string>NewTag</string>
   </property>
  </action>
  <action name="actionSaveTag">
   <property name="icon">
    <iconset resource="uiresource.qrc">
     <normaloff>:/root/filesave.png</normaloff>:/root/filesave.png</iconset>
   </property>
   <property name="text">
    <string>SaveTag</string>
   </property>
  </action>
  <action name="actionGlobalUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionGlobalRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToolbar</class>
   <extends>QWidget</extends>
   <header>qtoolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="uiresource.qrc"/>
 </resources>
 <connections/>
</ui>
